src/util/sort.js

```diff
@@ -0,0 +1,41 @@
+export const sortObject = (obj: any, key: string, inputKeyType: string = '', sortAscending: bool = true, keyChild: string): any => {
```

Comment:
why keep this method? remove it if not used (it should not be any need for this)